docs/tutorials/index.md

```diff
@@ -23,71 +23,63 @@ title: "Quickstart"
->
-> If needed, you can specify where to find Java using the environment variables `DRUID_JAVA_HOME` or `JAVA_HOME`. For more details run the verify-java script.
+The configuration profiles included with Druid range from the even smaller _Nano-Quickstart_ configuration (1 CPU, 4GB RAM) 
+to the _X-large_ configuration (64 CPU, 512GB RAM). For more information, see 
+[Single server deployment](operations/single-server). Alternatively, see [Clustered deployment](tutorials/cluster) for 
+information on deploying Druid services across clustered machines. 
 
-### Hardware
+The software requirements for the installation machine are:
 
-Druid includes several example [single-server configurations](../operations/single-server.md), along with scripts to
-start the Druid processes using these configurations.
+* Linux, Mac OS X, or other Unix-like OS (Windows is not supported)
+* Java 8, Update 92 or later (8u92+)
 
-If you're running on a small machine such as a laptop for a quick evaluation, the `micro-quickstart` configuration is
-a good choice, sized for a 4CPU/16GB RAM environment.
+> Druid officially supports Java 8 only. Support for later major versions of Java is currently in experimental status.
 
-If you plan to use the single-machine deployment for further evaluation beyond the tutorials, we recommend a larger
-configuration than `micro-quickstart`.
+> Druid relies on the environment variables `JAVA_HOME` or `DRUID_JAVA_HOME` to find Java on the machine. You can set 
+`DRUID_JAVA_HOME` if there is more than one instance of Java. To verify Java requirements for your environment, run the 
+`bin/verify-java` script.
 
-## Getting started
 
-[Download](https://www.apache.org/dyn/closer.cgi?path=/druid/{{DRUIDVERSION}}/apache-druid-{{DRUIDVERSION}}-bin.tar.gz)
-the {{DRUIDVERSION}} release.
+## Step 1. Install Druid
 
-Extract Druid by running the following commands in your terminal:
+After confirming the [requirements](#requirements), follow these steps: 
 
-```bash
-tar -xzf apache-druid-{{DRUIDVERSION}}-bin.tar.gz
-cd apache-druid-{{DRUIDVERSION}}
-```
+1. Download
+the [{{DRUIDVERSION}} release](https://www.apache.org/dyn/closer.cgi?path=/druid/{{DRUIDVERSION}}/apache-druid-{{DRUIDVERSION}}-bin.tar.gz).
+2. In your terminal, extract Druid and change directories to the distribution directory:
 
-In the package, you should find:
+   ```bash
+   tar -xzf apache-druid-{{DRUIDVERSION}}-bin.tar.gz
+   cd apache-druid-{{DRUIDVERSION}}
+   ```
+In the directory, you'll find `LICENSE` and `NOTICE` files and subdirectories for executable files, configuration files, sample data and more.
 
-* `LICENSE` and `NOTICE` files
-* `bin/*` - scripts useful for this quickstart
-* `conf/*` - example configurations for single-server and clustered setup
-* `extensions/*` - core Druid extensions
-* `hadoop-dependencies/*` - Druid Hadoop dependencies
-* `lib/*` - libraries and dependencies for core Druid
```

Comment:
In my personal opinion, I find this part useful when I was reading the original version.  It gives the reader a sense of what's included in the package.

Reply:
Thanks, I'll keep a note of this and see how this change goes over... I removed the list based on previous feedback (plus a directory and file were missing, so the content was out of date). 

I'm open to restoring though, especially if installation isn't covered elsewhere..I'd say it's a lot of detail for a quickstart, especially one that's gotten much longer, but the right level for an installation guide.